.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -24,9 +24,9 @@ body:
   attributes:
     label: Core Library
     options: 
-      - "@azure/msal or msal"
-      - "@azure/msal-browser"
+      - "@azure/msal-browser (v2)"
       - "@azure/msal-node"
+      - "@azure/msal or msal (v1)"
```

Comment:
I would if it is easier to have display names, e.g.

- MSAL.js v2 (@azure/msal-browser)
- MSAL.js v1 (@azure/msal or msal)
- MSAL React (@azure/msal-react)

etc

Reply:
I like that suggestion